ui/raidboss/data/03-hw/trial/zurvan-ex.ts

```diff
@@ -0,0 +1,234 @@
+
+export interface Data extends RaidbossData {
+  flameTarget?: string;
+  tetherBuddy?: string;
+  infiniteElement?: 'fire' | 'ice';
+}
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'ContainmentBayZ1T9Extreme,',
+  zoneId: ZoneId.ContainmentBayZ1T9Extreme,
+  timelineFile: 'zurvan-ex.txt',
+  timelineTriggers: [
+    {
+      id: 'ZurvanEX Metal Cutter',
+      regex: /Metal Cutter/,
+      beforeSeconds: 4,
+      suppressSeconds: 10,
+      response: Responses.tankCleave(),
+    },
+  ],
+  triggers: [
+    {
+      id: 'ZurvanEX Wave Cannon Avoid',
+      type: 'HeadMarker',
+      netRegex: { id: '000E' },
+      alarmText: (data, matches, output) => {
+        if (data.me === matches.target)
+          return output.waveCannonTarget!();
+      },
+      alertText: (data, matches, output) => {
+        if (!(data.me === matches.target))
+          return output.avoidWaveCannon!({ target: data.ShortName(matches.target) });
+      },
+      outputStrings: {
+        waveCannonTarget: {
+          en: 'Wave Cannon on YOU',
+        },
+        avoidWaveCannon: {
+          en: 'Away from ${target} -- Wave Cannon',
+        },
+      },
+    },
+    {
+      id: 'ZurvanEX Wave Cannon Stack',
+      type: 'StartsUsing',
+      netRegex: { id: '1C72', source: 'Zurvan' },
+      condition: Conditions.targetIsNotYou(), // The target is stunned during this mechanic
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'ZurvanEX Demon Claw',
+      type: 'StartsUsing',
+      netRegex: { id: '1C71', source: 'Zurvan' },
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.demonClawYou!(),
+      outputStrings: {
+        demonClawYou: {
+          en: 'Knockback from boss on YOU',
+        },
+      },
+    },
+    {
+      id: 'ZurvanEX Flaming Halberd',
+      type: 'HeadMarker',
+      netRegex: { id: '002C' },
+      condition: Conditions.targetIsYou(),
+      suppressSeconds: 1, // Don't spam people who run this solo.
+      response: Responses.spread(),
+    },
+    {
+      id: 'ZurvanEX Demonic Dive',
+      type: 'HeadMarker',
+      netRegex: { id: '003E' },
+      delaySeconds: 0.5,
+      alertText: (data, matches, output) => {
+        if (data?.flameTarget === data.me)
+          return;
+        if (data.flameTarget === undefined)
+          return output.unknownStackTarget!();
+        return output.stackOn!({ player: data.ShortName(matches.target) });
+      },
+      outputStrings: {
+        unknownStackTarget: Outputs.stackMarker,
+        stackOn: Outputs.stackOnPlayer,
+      },
+    },
+    {
+      id: 'ZurvanEX Cool Flame Call',
+      type: 'HeadMarker',
+      netRegex: { id: '0017' },
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.demonicSpread!(),
+      run: (data, matches) => data.flameTarget ??= matches.target,
+      outputStrings: {
+        demonicSpread: {
+          en: 'Spread -- Don\'t stack!',
```

Comment:
I think a spread response might be plenty here.  To me "spread" already implies "don't stack".

Reply:
It does, but this does follow conventions elsewhere in the repository of being explicit about it if there are both spread and stack mechanics resolving at once. Normally I would agree with you if this were a sequence of mechanics, something like Pantokrator in Omega Ultimate, but for a single mechanic like this I think it's useful to remind the user to watch for the stack.

Reply:
Where else do we do this? (Should those also just be spread??)

Reply:
[Alexander Prime](https://github.com/quisquous/cactbot/blob/4cda9ec2d52abf00dc1f6f4802f58d73fdc61c2e/ui/raidboss/data/03-hw/raid/a12n.ts#L71), [Hades](https://github.com/quisquous/cactbot/blob/4cda9ec2d52abf00dc1f6f4802f58d73fdc61c2e/ui/raidboss/data/05-shb/trial/hades.ts#L264) ,  and [Voidwalker](https://github.com/quisquous/cactbot/blob/4cda9ec2d52abf00dc1f6f4802f58d73fdc61c2e/ui/raidboss/data/05-shb/raid/e2n.ts#L134). Granted, Alexander Prime is my work too, so that's less compelling. (There's also a different reason not to stack there, so it's maybe not an exact analogy.)

Reply:
Ok, sounds good! I don't feel that strongly here.